src/collections/_documentation/platforms/node/serverless.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Serverless
+sidebar_order: 7
+---
+
+## AWS Lambda
+
+To set up Sentry error logging for a Lambda function, build a wrapper.
+
+```javascript
+    "use strict";
```

Comment:
Code looks good, although please pass it through the code style formatter. You can use this one and paste your code example directly - https://prettier.io/playground/

Reply:
Ohhh nice. Thanks!